docs/linking/1-discovery.puml

```diff
@@ -21,8 +21,6 @@ activate PISP
 
```

Comment:
@lewisdaly `AccountsIDPutResponse` response shouldn't be an array but object with `accounts` property 

In api-snippets is the old (end potentially dangerous) version: https://github.com/mojaloop/api-snippets/blob/master/thirdparty/openapi3/components/schemas/AccountsIDPutResponse.yaml

Reply:
Ok, yeah we need to fix this in the api draft and our sequences. I'll make a note to follow it up.